torchvision/transforms/transforms.py

```diff
@@ -527,6 +527,69 @@ def __repr__(self):
         return self.__class__.__name__ + '(p={})'.format(self.p)
 
 
+class RandomPerspective(object):
+    """Performs Perspective transformation of the given PIL Image randomly with a given probability.
+
+    Args:
+        p (float): probability of the image being perspectively transformed. Default value is 0.5
+        interpolation : Default- Image.BICUBIC
+    """
+
+    def __init__(self, interpolation=Image.BICUBIC, p=0.5):
```

Comment:
I think we could expose a few more parameters here, to give more control to the user of the range of possible transformations maybe?

Reply:
@fmassa i didnt get you on this.
Can you please  elaborate on this.

Reply:
I'm not clear on all the aspects here, but I was thinking if there would be a few meaningful arguments that we could pass that would restrict the range of possible transformations?
For example, if we were only to consider rotation on a 2d plane, to pass a min/max angles?

I think it's not trivial to come up with some meaningful values, but I was raising this point to gather some feedback